### PR TITLE
Comment out AFE amplitude events temporarily

### PR DESCRIPTION
**Pull request by hannahbergam**
Currently, the amplitude event logger is causing the entire AFE submission form to fail. This PR removes all AFE logging for now (the events are also captured in firehose) until we can figure out what is going on with the API key. 

I opted for this approach because the Amplitude implementation was not from just one PR. This way, the PR can have a clean revert when we figure out why there is an issue with the API key. 

## Warning!!

We have entered Pixel Lock for Hour of Code!

Computer Science Education Week will be happening from Dec 4 - Dec 10. Alongside this event, we will
be launching our new Hour of Code activity. Please consider any risk introduced in this PR that
could affect Dance Lab, instructions, saving and logging student progress, caching, or anything
related to the Hour of Code activities, old or new. Even small changes, such as a different button
color, are considered significant during this time. If this change will affect the new Hour of Code
activity in any way, join the morning change review to get your changes approved prior to merging.
Reach out to the Student Labs team for more details!

## Links

- spec: []()
- jira ticket: https://codedotorg.atlassian.net/browse/ACQ-1212

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/sites/code.org/pages/views/amazon_future_engineer_eligibility.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import firehoseClient from '@cdo/apps/lib/util/firehose';
-import analyticsReporter from '@cdo/apps/lib/util/AnalyticsReporter';
-import {EVENTS} from '@cdo/apps/lib/util/AnalyticsConstants';
+//import analyticsReporter from '@cdo/apps/lib/util/AnalyticsReporter';
```
Comment (bethanyaconnor) [inline apps/src/sites/code.org/pages/views/amazon_future_engineer_eligibility.js:2]: Could you add a TODO comment here, ideally linking to a jira ticket? I think you could skip Drone for that commit (assuming this one passes of course)

**Review by bethanyaconnor — APPROVED** [review]
Makes sense!

As an aside, can we repro this issue on staging? That should be logging to Amplitude (in a different project)

**Comment by hannahbergam** [comment]
> Makes sense!
> 
> As an aside, can we repro this issue on staging? That should be logging to Amplitude (in a different project)

Yes- this is also happening on staging. I'm actually able to see some part of the issue locally too (was able to see the api request returning a null value on Wednesday) but wasn't able to find a fix quickly. Hopefully this will buy me a little bit more time!

File: apps/src/sites/code.org/pages/views/amazon_future_engineer_eligibility.js (hunk 1 above)
Comment (hannahbergam) [reply apps/src/sites/code.org/pages/views/amazon_future_engineer_eligibility.js:2]: (done- good call)

**Pull request merged**